src/client.rs

```diff
@@ -121,7 +144,7 @@ impl Client {
     /// # movies.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     /// # });
     /// ```
-    pub fn multi_search(&self) -> MultiSearchQuery {
+    pub fn multi_search(&self) -> MultiSearchQuery<Http> {
```

Comment:
Is it possible to hide the http client here so that the user does not have to provide it if they use ishac?

Reply:
I have thought about it a bit, but it seems difficult due to the presence of the Client in the `MultiSearchQuery` structure🤔
If the use case of `MultiSearchQuery` is only to be called as a method from the `Client` structure, I don't think there is that much of a problem. 
However, if you are going to implement `MultiSearchQuery` directly, I think it would be better to modify it.

Reply:
Done